src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java

```diff
@@ -43,6 +44,12 @@
     @Inject
     public Injector injector;
 
+    /**
+     * The controller that starts/stops Jenkins
+     */
+    @Inject
+    public JenkinsController controller;
```

Comment:
My idea was to put it in the top level element so that it could be used everywhere if needed. Moved it to `Job` anyway.